test/e2e/handler/bonding_default_interface_test.go

```diff
@@ -28,12 +28,12 @@ import (
 	"github.com/nmstate/kubernetes-nmstate/test/e2e/policy"
 )
 
-// TODO: When https://bugzilla.redhat.com/show_bug.cgi?id=1906307 is resolved, add firstSecondaryNic to the bond again
 func boundUpWithPrimaryAndSecondary(bondName string) nmstate.State {
 	return nmstate.NewState(fmt.Sprintf(`interfaces:
   - name: %s
     type: bond
     state: up
+	copy-mac-from: %s
```

Comment:
Could we extend our bond examples with this too? It is a useful attribute, would be nice to make people aware of it via user-guide

Reply:
Done, added the copy-mac-from to the bond we create at example.